### PR TITLE
chore: update docs for each usecase

### PR DESCRIPTION
**Pull request by danielbdias**
This PR adds a README on each example explaining a who to use the Github Action on each specific case.

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: examples/tracetest-cli-and-agent-with-tracetest-io/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Running Trace-based tests using Tracetest Agent and CLI with Tracetest.io
+
+To add Tracetest to your GitHub pipeline and use [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference) and [Tracetest Agent](https://docs.tracetest.io/concepts/agent) to run tests of [Tracetest.io](https://docs.tracetest.io/), you need just to add the following snippet in your workflow as a step:
```
Comment (adnanrahic) [inline examples/tracetest-cli-and-agent-with-tracetest-io/README.md]:
```suggestion
To add Tracetest to your GitHub pipeline and use [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference) and [Tracetest Agent](https://docs.tracetest.io/concepts/agent) to run tests on [Tracetest](https://app.tracetest.io/), you need to add the following snippet in your workflow as a step:
```

File: examples/tracetest-cli-and-agent-with-tracetest-io/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+```yaml
+- name: Configure Tracetest CLI and Agent
+  uses: kubeshop/tracetest-github-action@v1
+  with:
+    token: "{your tracetest token}" # See https://docs.tracetest.io/concepts/environment-tokens for more details
+    apiKey: "{your tracetest agent apiKey}" # See https://docs.tracetest.io/concepts/agent for more details
+```
+
+You can get your Tracetest Token, on Tracetest.io by following the directions described [here](https://docs.tracetest.io/concepts/environment-tokens). The Tracetest Agent APIKey can be seen on `Agent` tab on `Settings` page.
```
Comment (adnanrahic) [inline examples/tracetest-cli-and-agent-with-tracetest-io/README.md]:
```suggestion
You can get your Tracetest Token from the Tracetest settings page by following the directions described [here](https://docs.tracetest.io/concepts/environment-tokens). The Tracetest Agent APIKey can be seen on the `Agent` tab on `Settings` page.
```

File: examples/tracetest-cli-and-agent-with-tracetest-io/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+A full example of usage can be seen on [tracetest-cli-and-agent-with-tracetest-io.yml](./tracetest-cli-and-agent-with-tracetest-io.yml) file located in this folder.
```
Comment (adnanrahic) [inline examples/tracetest-cli-and-agent-with-tracetest-io/README.md]:
```suggestion
Here's a full example of how to use it: [tracetest-cli-and-agent-with-tracetest-io.yml](./tracetest-cli-and-agent-with-tracetest-io.yml) 
```

File: examples/tracetest-cli-using-custom-config/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Running Trace-based tests using Tracetest CLI with custom config on Tracetest.io
+
+To add Tracetest to your GitHub pipeline and use [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference) to run tests of [Tracetest.io](https://docs.tracetest.io/) using a customized config, you need just to add the following snippet in your workflow as a step:
```
Comment (adnanrahic) [inline examples/tracetest-cli-using-custom-config/README.md]:
```suggestion
To add Tracetest to your GitHub pipeline and use [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference) to run tests on [Tracetest](https://app.tracetest.io/) using a customized config, you need to add the following snippet in your workflow as a step:
```

File: examples/tracetest-cli-using-custom-config/README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+```yaml
+- name: Configure Tracetest CLI
+  uses: kubeshop/tracetest-github-action@v1
+  with:
+    configFile: "scheme: https\nendpoint: your.tracetest.url"
+```
+
+Where the `configFile` parameter is the contents of your CLI config file, usually located on `~/.tracetest/config.yml`.
```
Comment (adnanrahic) [inline examples/tracetest-cli-using-custom-config/README.md]:
```suggestion
Where the `configFile` parameter is the content of your CLI config file, usually located in `~/.tracetest/config.yml`.
```

File: examples/tracetest-cli-using-custom-config/README.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+A full example of usage can be seen on [tracetest-cli-using-custom-config.yml](./tracetest-cli-using-custom-config.yml) file located in this folder.
```
Comment (adnanrahic) [inline examples/tracetest-cli-using-custom-config/README.md]:
```suggestion
Here's a full example of how to use it: [tracetest-cli-using-custom-config.yml](./tracetest-cli-using-custom-config.yml)
```

File: examples/tracetest-cli-with-tracetest-core/README.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+# Running Trace-based tests using Tracetest CLI and Tracetest Core
+
+To add Tracetest to your GitHub pipeline and use [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference) to run tests of your own [Tracetest Core server](https://docs.tracetest.io/core/getting-started/installation#install-the-tracetest-server), you need just to add the following snippet in your workflow as a step:
```
Comment (adnanrahic) [inline examples/tracetest-cli-with-tracetest-core/README.md]:
```suggestion
To add Tracetest to your GitHub pipeline and use [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference) to run tests on your own [Tracetest Core Server](https://docs.tracetest.io/core/getting-started/installation#install-the-tracetest-server), you need to add the following snippet in your workflow as a step:
```

File: examples/tracetest-cli-with-tracetest-core/README.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+```yaml
+- name: Configure Tracetest CLI
+  uses: kubeshop/tracetest-github-action@v1
+  with:
+    endpoint: "{URL to your Tracetest Core Server}"
+```
+
+The `endpoint` parameter is the base address where your Tracetest Core Server is installed.
+
+A full example of usage can be seen on [tracetest-cli-with-tracetest-core.yml](./tracetest-cli-with-tracetest-core.yml) file located in this folder.
```
Comment (adnanrahic) [inline examples/tracetest-cli-with-tracetest-core/README.md]:
```suggestion
Here's a full example of how to use it: [tracetest-cli-with-tracetest-core.yml](./tracetest-cli-with-tracetest-core.yml)
```

File: examples/tracetest-cli-with-tracetest-io/README.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+# Running Trace-based tests using Tracetest CLI with Tracetest.io
+
+To add Tracetest to your GitHub pipeline and use [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference) to run tests of [Tracetest.io](https://docs.tracetest.io/), you need just to add the following snippet in your workflow as a step:
```
Comment (adnanrahic) [inline examples/tracetest-cli-with-tracetest-io/README.md]:
```suggestion
To add Tracetest to your GitHub pipeline and use [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference) to run tests on [Tracetest](https://app.tracetest.io/), you need to add the following snippet in your workflow as a step:
```

File: examples/tracetest-cli-with-tracetest-io/README.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+
+```yaml
+- name: Configure Tracetest CLI
+  uses: kubeshop/tracetest-github-action@v1
+  with:
+    token: "{your tracetest token}" # See https://docs.tracetest.io/concepts/environment-tokens for more details
+```
+
+You can get your Tracetest Token, on Tracetest.io by following the directions described [here](https://docs.tracetest.io/concepts/environment-tokens).
```
Comment (adnanrahic) [inline examples/tracetest-cli-with-tracetest-io/README.md]:
```suggestion
You can get your Tracetest Token, on the Tracetest settings page by following the directions described [here](https://docs.tracetest.io/concepts/environment-tokens). 
```

File: examples/tracetest-cli-with-tracetest-io/README.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+```yaml
+- name: Configure Tracetest CLI
+  uses: kubeshop/tracetest-github-action@v1
+  with:
+    token: "{your tracetest token}" # See https://docs.tracetest.io/concepts/environment-tokens for more details
+```
+
+You can get your Tracetest Token, on Tracetest.io by following the directions described [here](https://docs.tracetest.io/concepts/environment-tokens). 
+
+A full example of usage can be seen on [tracetest-cli-with-tracetest-io.yml](./tracetest-cli-with-tracetest-io.yml) file located in this folder.
```
Comment (adnanrahic) [inline examples/tracetest-cli-with-tracetest-io/README.md]:
```suggestion
Here's a full example of how to use it: [tracetest-cli-with-tracetest-io.yml](./tracetest-cli-with-tracetest-io.yml)
```

**Review by adnanrahic — CHANGES_REQUESTED** [review]
Awesome work Daniel!
Added a few small suggestions.

File: examples/tracetest-cli-and-agent-with-tracetest-io/README.md [hunk 12]
```diff
@@ -0,0 +1,15 @@
+# Running Trace-based tests using Tracetest Agent and CLI with Tracetest.io
```
Comment (jfermi) [inline examples/tracetest-cli-and-agent-with-tracetest-io/README.md]:
```suggestion
# Running Trace-based Tests using Tracetest Agent and CLI with Tracetest.io
```

File: examples/tracetest-cli-using-custom-config/README.md [hunk 13]
```diff
@@ -0,0 +1,14 @@
+# Running Trace-based tests using Tracetest CLI with custom config on Tracetest.io
```
Comment (jfermi) [inline examples/tracetest-cli-using-custom-config/README.md]:
```suggestion
# Running Trace-based Tests using Tracetest CLI with Custom Config on Tracetest.io
```

File: examples/tracetest-cli-with-tracetest-core/README.md [hunk 14]
```diff
@@ -0,0 +1,14 @@
+# Running Trace-based tests using Tracetest CLI and Tracetest Core
```
Comment (jfermi) [inline examples/tracetest-cli-with-tracetest-core/README.md]:
```suggestion
# Running Trace-based Tests using Tracetest CLI and Tracetest Core
```

File: examples/tracetest-cli-with-tracetest-io/README.md [hunk 15]
```diff
@@ -0,0 +1,14 @@
+# Running Trace-based tests using Tracetest CLI with Tracetest.io
```
Comment (jfermi) [inline examples/tracetest-cli-with-tracetest-io/README.md]:
```suggestion
# Running Trace-based Tests using Tracetest CLI with Tracetest.io
```

File: examples/tracetest-cli-with-tracetest-io/README.md [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+To add Tracetest to your GitHub pipeline and use [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference) to run tests on [Tracetest](https://app.tracetest.io/), you need to add the following snippet in your workflow as a step:
+
+```yaml
+- name: Configure Tracetest CLI
+  uses: kubeshop/tracetest-github-action@v1
+  with:
+    token: "{your tracetest token}" # See https://docs.tracetest.io/concepts/environment-tokens for more details
+```
+
+You can get your Tracetest Token, on the Tracetest settings page by following the directions described [here](https://docs.tracetest.io/concepts/environment-tokens).
```
Comment (jfermi) [inline examples/tracetest-cli-with-tracetest-io/README.md]:
```suggestion
You can get your Tracetest Token on the Tracetest settings page by following the directions described [here](https://docs.tracetest.io/concepts/environment-tokens). 
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Review by adnanrahic — APPROVED** [review]

**Pull request merged**